pysat/instruments/ucar_tiegcm.py

```diff
@@ -4,21 +4,20 @@
 (Thermosphere Ionosphere Electrodynamics General Circulation Model) model.
 TIEGCM file is a netCDF file with multiple dimensions for some variables.
 
-Parameters
+Properties
 ----------
-platform : string
+platform
     'ucar'
-name : string
+name
     'tiegcm'
-tag : string
+tag
     None supported
-sat_id : string
+sat_id
     None supported
 
+
 Note
 ----
-::
-
     Loads into xarray format.
```

Comment:
Is this necessary?

Reply:
Probably not, now that we have multiple xarray instruments